File: projects/m1/033-shipping-calculator/solutions/RoccoFigliamonti/shipping_calculator.py
"""# Shipping Calculator

An online retailer provides express shipping for many of its items at a rate of €10.99 for the first item in an order, and €2.99 for each subsequent item in the same order. 

Write a function that takes the number of items in the order as its only parameter. 

Return the shipping charge for the order as the function’s result. 

Include a main program that reads the number of items purchased from the user and displays the shipping charge."""

def shipping_charge(num_of_items):
    """This program calcs the shipping charge for a given number of items,
    considering the first item charge of 10.99 and 2.99 for each subsequent item"""
    fee_per_item = 2.99
    if num_of_items==1:
        total_charge = 10.99
    else:
        total_charge = (num_of_items-1)*fee_per_item + 10.99
    return round(total_charge,2)
    

print(f"The total charge is {shipping_charge(5)} $")

File: projects/m1/033-shipping-calculator/solutions/alessiacaporaso/shipping-calculator.js
// €10.99 rate for the first item
// €2.99 rate for item on the same order
// function takes the number of items in the order as its only parameter
const numberElement = parseInt(prompt('Enter the number of item in an order'))
function calcShippingCare(item){
    // let shippingCharge = 0;
    // if(item === 1){
    //     shippingCharge = 10.99
    // } else {
    const shippingCharge = (2.99 * (item - 1) + 10.99).toFixed(2)
    // }
    return shippingCharge
}

console.log(`Shipping costs for ${numberElement} items are €${calcShippingCare(numberElement)}`)
// calcShippingCare(1)
// calcShippingCare(3)
// calcShippingCare(20)